default/scripting/species/SP_ANCIENT_GUARDIANS.focs.txt

```diff
@@ -0,0 +1,90 @@
+Species
+    name = "SP_ANCIENT_GUARDIANS"
+    description = "SP_ANCIENT_GUARDIANS_DESC"
+    gameplay_description = "SP_ANCIENT_GUARDIANS_GAMEPLAY_DESC"
+
+    tags = [ "ROBOTIC" "NO_INDUSTRY" "NO_RESEARCH" "AVERAGE_SUPPLY" "DESTROYED_ON_CONQUEST"]
+
+    foci = [
+        [[HAS_INDUSTRY_FOCUS]]
+        [[HAS_RESEARCH_FOCUS]]
+        [[HAS_GROWTH_FOCUS]]
+        [[HAS_ADVANCED_FOCI]]
```

Comment:
If we use Protection that would make complete sense but would mean if we change what protection does down the line we have to reasses the numbers here. Having the focuses there makes little sense as they should, in theory, never be used, but removing them isn't a priority I think.

When we do more work on them setting it to protection only and fiddling with the numbers can work, but my test games reveal the AI team need to make tweaks before we make them any more annoying—my neighbour in my current game has over 100 troops in orbit around one planet but no warship in sight.